gemini/evaluation/get_started_with_genai_model_eval_service.ipynb

```diff
@@ -824,7 +835,32 @@
    },
    "outputs": [],
    "source": [
-    "llm1_model = GenerativeModel(\"gemini-pro\")\n",
+    "generation_config = {\n",
```

Comment:
Can you use the `GenerationConfig` class?

Reply:
done